packages/opentelemetry-semantic-conventions/package.json

```diff
@@ -29,6 +29,9 @@
     "node": ">=8.0.0"
   },
   "files": [
+    "build/esm/**/*.js",
```

Comment:
Doesn't this double the size on disk for nodejs users?

Reply:
It does, we agreed before that its worth (when discussing about the API build)